synth.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Google LLC
```

Comment:
This is a bit confusing if it's in the root of the repo.

Do we even need it, given that this isn't an autogenerated repo?

Reply:
removed